Ports/RISC-V/RV32/GCC/os_cpu_a.S

```diff
@@ -153,60 +168,109 @@ OSStartHighRdy:
 #              the result of an interrupt.  This function triggers a synchronous software interrupt by
 #              writing into the MSIP register
 #********************************************************************************************************
+OSIntCtxSw:
```

Comment:
Check the comments above this line, and change them to match the actual code.
e.g we use ecall and not MSIP

Also we keep OSIntCtxSw as blank since the context switch action after interrupt is done just before exiting the interrupt handler